web/src/app/services/IntegrationKeyList.tsx

```diff
@@ -71,27 +75,24 @@ export function IntegrationKeyDetails(props) {
       {props.type === 'email' && (
         <RequireConfig
           configID='Mailgun.Enable'
-          else='Email integration keys are currently disabled.'
+          else={<div>Email integration keys are currently disabled.</div>}
```

Comment:
```suggestion
          else={
            <React.Fragment>
              Email integration keys are currently disabled.
            </React.Fragment>
          }
```
Looks like the `<div>` here is throwing an error

<img width="464" alt="Screen Shot 2022-09-08 at 9 44 22 AM" src="https://user-images.githubusercontent.com/38022260/189152575-1ff5fa3f-004b-4d0d-89db-f677cd40cf3a.png">